bin/init.sh

```diff
@@ -8,6 +8,14 @@ rm -f django.db
 
 # run database migrations
 
+if [[ ${DJANGO_LOAD_SAMPLE_DATA:-true} = false ]]; then
+    if [[ -d ${DJANGO_MIGRATIONS_DIR:-false} ]]; then
+        cp ${DJANGO_MIGRATIONS_DIR}/0002_*.py ./benefits/core/migrations/
```

Comment:
Sorry one more minor thing... can we print the directory here, or the `ls` of the files that will be copied from it? Could help with debugging in Azure later.

Reply:
Sure

Reply:
Like `Copying migrations from ${DJANGO_MIGRATIONS_DIR}` or something?